io/subsystem/src/main/java/org/wildfly/extension/io/BufferPoolResourceDefinition.java

```diff
@@ -42,79 +42,70 @@
 
-    private static final int defaultBufferSize;
-    private static final int defaultBuffersPerRegion;
-    private static final boolean defaultDirectBuffers;
+    static final ModelNode DEFAULT_BUFFER_SIZE;
+    static final ModelNode DEFAULT_BUFFERS_PER_REGION;
+    static final ModelNode DEFAULT_DIRECT_BUFFERS;
 
     static {
+        int defaultBufferSize;
+        int defaultBuffersPerRegion;
+        boolean defaultDirectBuffers;
+
         long maxMemory = Runtime.getRuntime().maxMemory();
-        //smaller than 64mb of ram we use 512b buffers
+        // smaller than 64mb of ram we use 512b buffers
         if (maxMemory < 64 * 1024 * 1024) {
-            //use 512b buffers
+            // use 512b buffers
             defaultDirectBuffers = false;
             defaultBufferSize = 512;
             defaultBuffersPerRegion = 10;
         } else if (maxMemory < 128 * 1024 * 1024) {
-            //use 1k buffers
+            // use 1k buffers
             defaultDirectBuffers = true;
             defaultBufferSize = 1024;
             defaultBuffersPerRegion = 10;
         } else {
-            //use 16k buffers for best performance
-            //as 16k is generally the max amount of data that can be sent in a single write() call
+            // use 16k buffers for best performance
+            // as 16k is generally the max amount of data that can be sent in a single write() call
             defaultDirectBuffers = true;
             defaultBufferSize = 1024 * 16;
             defaultBuffersPerRegion = 20;
         }
+        DEFAULT_BUFFER_SIZE = new ModelNode(defaultBufferSize);
+        DEFAULT_BUFFERS_PER_REGION = new ModelNode(defaultBuffersPerRegion);
+        DEFAULT_DIRECT_BUFFERS = new ModelNode(defaultDirectBuffers);
     }
 
-    static final SimpleAttributeDefinition BUFFER_SIZE = new SimpleAttributeDefinitionBuilder(Constants.BUFFER_SIZE, ModelType.INT, true)
-            .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
-            .setAllowExpression(true)
-            .setValidator(new IntRangeValidator(1, true, true))
-            .build();
-    static final SimpleAttributeDefinition BUFFER_PER_SLICE = new SimpleAttributeDefinitionBuilder(Constants.BUFFER_PER_SLICE, ModelType.INT, true)
-            .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
-            .setAllowExpression(true)
-            .setValidator(new IntRangeValidator(1, true, true))
-            .build();
-    static final SimpleAttributeDefinition DIRECT_BUFFERS = new SimpleAttributeDefinitionBuilder(Constants.DIRECT_BUFFERS, ModelType.BOOLEAN, true)
-            .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
-            .setAllowExpression(true)
-            .build();
-
-
-    /*<buffer-pool name="default" buffer-size="1024" buffers-per-slice="1024"/>*/
+    static final SimpleAttributeDefinition BUFFER_SIZE = new SimpleAttributeDefinitionBuilder(Constants.BUFFER_SIZE,
```

Comment:
putting everything on the same line makes it hard to read and mantain.

Could you revert to the previous formatting, please?

Reply:
I think the Eclipse formatter screwed me on this one. I'll fix it indeed, thanks!